### DISCUSSION TITLE
Instazlition failed.

### DISCUSSION THREAD

**Discussion by aakashrio** [rancher-sandbox/rancher-desktop#7388] [Q&A]
Hi team

Initialization failed while installing Rancher Desktop using cmd (as administrator). I used the silent key "**/quiet /norestart**". Why is this happening? I have attached the install log files for you, please check and tell me why it's happening.

I need more information. I received exit code 1603. Can you please help me solve this issue?
[install.log](https://github.com/user-attachments/files/16792115/install.log)

Thanks in advance!

**Comment by mook-as** [comment]
The log says:
> time="2024-08-29T09:57:01+05:30" level=info msg="--install: unrecognized option: no-distribution"

That sounds like #6415

As a workaround, please install WSL2 manually before installing Rancher Desktop.  Please see [Microsoft docs](https://learn.microsoft.com/en-us/windows/wsl/install) for details.

Note that the logs also say:
> time="2024-08-29T09:55:37+05:30" level=info msg="WSL install state: Version=0.0.0.0 kernel=0.0.0.0 (inbox, outdated-kernel)"

So it sounds like you may have WSL1 installed; note that we require WSL2.

What version of Windows are you running? `msiexec` was never rebuilt, so it still claims that you're running Windows 8.1, which is probably false (because you have _some_ kind of WSL).

**Reply by aakashrio** [reply]
Thanks for the information.
